experimental/packages/exporter-trace-otlp-grpc/package.json

```diff
@@ -48,6 +48,7 @@
   },
   "devDependencies": {
     "@babel/core": "7.16.0",
+    "@grpc/proto-loader": "^0.7.3",
```

Comment:
Is `@grpc/proto-loader` needed in the dev dependencies even `@grpc/grpc-js` is a dependency?
I thought `@grpc/grpc-js` ensures that the correct/matching version of `@grpc/proto-loader` is installed

Reply:
if we depend on a package directly then it should be in the package.json. Even if it is transitively required by another package npm handles the dedupe when required.